src/pages/repository.html

```diff
@@ -73,8 +73,15 @@ <h1>{{pageTitle}}</h1>
                                 </div>
                                 <input class="form-control" id="file" type="file" name="file" style="display: none;"/>
                             </div>
-
+                            <div class="form-group col-xs-2 col-sm-2 col-md-2">
+                                <button id="testConnection" ng-click="validateOntopPropertiesConnection()"
```

Comment:
It might be better to disable this button when no file is selected.

Reply:
it is disabled when no file is selected - ng-disabled="!isOntopRepoFileUploaded() && editRepoPage"